packages/base/src/panelview/components/identify-panel/IdentifyPanel.tsx

```diff
@@ -0,0 +1,195 @@
+    controlPanelModel?.jGISModel
+  );
+
+  const featuresRef = useRef(features);
+  /**
+   * Update the model when it changes.
+   */
+  controlPanelModel?.documentChanged.connect((_, widget) => {
+    setJgisModel(widget?.context.model);
+  });
+
+  // Reset state values when current widget changes
+  useEffect(() => {
+    const handleCurrentChanged = () => {
+      if (tracker.currentWidget?.id === widgetId) {
+        return;
+      }
+
+      if (tracker.currentWidget) {
+        setWidgetId(tracker.currentWidget.id);
+      }
+      setFeatures({});
+      setVisibleFeatures({});
+    };
+    tracker.currentChanged.connect(handleCurrentChanged);
+
+    return () => {
+      tracker.currentChanged.disconnect(handleCurrentChanged);
+    };
+  }, []);
+
+  useEffect(() => {
+    featuresRef.current = features;
+  }, [features]);
+
+  useEffect(() => {
+    const handleClientStateChanged = (
+      sender: IJupyterGISModel,
+      clients: Map<number, IJupyterGISClientState>
+    ) => {
+      const remoteUserId = jgisModel?.localState?.remoteUser;
+
+      // If following a collaborator
+      if (remoteUserId) {
+        const remoteState = clients.get(remoteUserId);
+        if (remoteState) {
+          if (remoteState.user?.username !== remoteUser?.username) {
+            setRemoteUser(remoteState.user);
+          }
+
+          setFeatures(remoteState.identifiedFeatures?.value ?? {});
+        }
+        return;
+      }
+
+      // If not following a collaborator
+      const identifiedFeatures =
+        jgisModel?.localState?.identifiedFeatures?.value;
+
+      if (!identifiedFeatures) {
+        setFeatures({});
+        return;
+      }
+
+      if (
+        jgisModel.isIdentifying &&
+        featuresRef.current !== identifiedFeatures
+      ) {
+        setFeatures(identifiedFeatures);
+      }
+    };
+
+    jgisModel?.clientStateChanged.connect(handleClientStateChanged);
+
+    return () => {
+      jgisModel?.clientStateChanged.disconnect(handleClientStateChanged);
+    };
+  }, [jgisModel]);
+
+  const toggleFeatureVisibility = (index: number) => {
+    setVisibleFeatures(prev => ({
+      ...prev,
+      [index]: !prev[index]
+    }));
+  };
+
+  return (
+    <div
+      className="jgis-identify-wrapper"
+      style={{
+        border: jgisModel?.localState?.remoteUser
+          ? `solid 3px ${remoteUser?.color}`
+          : 'unset'
+      }}
+    >
+      {features &&
+        Object.values(features).map((feature, featureIndex) => (
+          <div key={featureIndex} className="jgis-identify-grid-item">
+            <div
+              className="jgis-identify-grid-item-header"
+              onClick={() => toggleFeatureVisibility(featureIndex)}
+            >
+              <LabIcon.resolveReact
+                icon={caretDownIcon}
+                className={`jp-gis-layerGroupCollapser${visibleFeatures[featureIndex] ? ' jp-mod-expanded' : ''}`}
```

Comment:
What do you think of always having Feature 1 expanded after doing an identify interaction, mirroring QGIS' behavior? There's never a case where the user would want all the results collapsed, I think.